src/renderer/Components/CreatingPackage/CreatingOrder/BookmarkToolbar/BookmarkList/index.jsx

```diff
@@ -17,12 +17,33 @@ function BookmarkList({ bookmarks, closeBookmarkList }) {
       {bookmarks.map((bookmark, index) => (
         <button
           key={bookmark._id}
-          className="button-base-blue"
+          className="button-gray-light block text-left"
           onClick={() => {
             applyBookmark(index);
           }}
         >
-          {bookmark.action}
+          <span className="mr-2 font-bold text-black">
+            {index + 1}
+            {". "}
+          </span>
+          <span className="text-lg font-light">
+            {`"${bookmark.attachmentName}" 을(를) `}
+          </span>
+          {bookmark.sourcePath && (
+            <span className="text-lg font-light">
+              {bookmark.sourcePath}에서{" "}
+            </span>
+          )}
+          <span className="text-lg font-light">
+            {` "${bookmark.executionPath}" `}
+            {bookmark.action === "이동하기" ? "(으)로 " : "에서 "}
+          </span>
+          {bookmark.editingName && (
+            <span className="text-lg font-light">
+              {bookmark.editingName} (으)로{" "}
```

Comment:
```js
{`"${bookmark.attachmentName}" 을(를) `}
```
위에서 썼던 코드 처럼 해당 코드도 일관성 있게 아래처럼 바꾸면 어떨까요? 🤔
```suggestion
              {`"${bookmark.editingName}" (으)로 `}
```

Reply:
그렇네요!😃 일관성 있게 변경해보도록 하겠습니다!